consensus/poet/sgx/tests/test_sgx/path.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Intel Corporation
```

Comment:
It's part of removing the dependency on the validator code.  Items from the validator source tree were replaced by the classes added in this PR.  The shim was created from the validator's perspective (i.e. replacing the validator's side of the surface) and PoET was (largely) treated as a black box.

Reply:
Per discussion there's the potential for some cross package dependencies that we want to avoid. Maybe as future work someone can take a look at why the tests need the config paths.